app/services/title_builder.rb

```diff
@@ -21,7 +21,8 @@ def self.build(titles)
   # rubocop:disable Metrics/PerceivedComplexity
   # @param [Array<Cocina::Models::StructuredValue>] structured_values - the individual pieces of a structuredValue to be combined
   # @param [Integer] the length of the non_sorting_characters
-  # @returns [String] the title value from combining the pieces of the structured_values according to type and order of occurrence
+  # @returns [String] the title value from combining the pieces of the structured_values according to type and order of occurrence,
```

Comment:
It looks like 6 of the 41 uses of `@return(s)` in DIA use `@returns` instead of `@return` and I believe the latter is more correct per YARD documentation. Would you consider changing `@returns` to `@return` here? Alternatively, I'd be happy to put in a follow-up PR to replace/fix all of these for consistency and increased C O R R E C T N E S S. Up to you!

Reply:
i will fix.